libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -242,8 +236,8 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @Values{Rover}: 202:Pitch
     // @Values{Rover}: 207:MainSail
     // @Values{Rover, Plane}:  208:Flap
-    // @Values{Plane}: 209: Forward Throttle
-    // @Values{Plane}: 210: Airbrakes
+    // @Values{Plane}: 209:VTOL Forward Throttle
+    // @Values{Plane}: 210:Airbrakes
```

Comment:
no big deal but why remove this space here but not in other items?  (e.g. "150: Mode Cruise")